main_server/local_server/server_admin/c_comdef_admin_main_console.class.php

```diff
@@ -1682,6 +1682,22 @@ public function return_single_meeting_basic_template()
                     $ret .= '</span>';
                     $ret .= '<div class="clear_both"></div>';
                 $ret .= '</div>';
+
+            if (isset($this->my_localized_strings['meeting_time_zones_enabled'])) {
+                $ret .= '<div class="bmlt_admin_one_line_in_a_form clear_both">';
+                $ret .= '<span class="bmlt_admin_med_label_right">'.htmlspecialchars($this->my_localized_strings['comdef_server_admin_strings']['meeting_editor_screen_meeting_time_zone_label']).'</span>';
+                $ret .= '<span class="bmlt_admin_value_left">';
+                $ret .= '<select id="bmlt_admin_single_meeting_editor_template_meeting_time_zone_select_input" class="bmlt_admin_single_meeting_editor_template_meeting_time_zone_select_input">';
+                $ret .= '<option value=""></option>';
+                foreach (DateTimeZone::listIdentifiers(DateTimeZone::ALL) as $tz) {
```

Comment:
I think we should store a static list of these in the code, because the return value of this function could differ between PHP versions. I there is an official tzinfo database, we shove those into an array in the root server code.

Reply:
You can probably store the list in `my_localized_strings` or something, or as a default setting

Reply:
this totes made sense, added to localized strings.